modules/freewheelSSPBidAdapter.js

```diff
@@ -223,6 +223,24 @@ export const spec = {
       componentId: getComponentId(currentBidRequest.params.format)
     };
 
+    // Add GDPR flag and consent string
+    if (bidderRequest.gdprConsent) {
+      requestParams._fw_gdpr_consent = bidderRequest.gdprConsent.consentString;
+
+      if (typeof bidderRequest.gdprConsent.gdprApplies === 'boolean') {
+        requestParams._fw_gdpr = bidderRequest.gdprConsent.gdprApplies;
+      }
+    }
+
+    var vastParams = currentBidRequest.params.vastUrlParams;
+    if (typeof vastParams === 'object') {
+      for (key in vastParams) {
```

Comment:
Update - I thought this was addressed in the commit, but it seems it wasn't.  The LGTM check noted this `key` variable isn't properly declared (missing a `let`).  Could you update this spot?

Reply:
Done.